homeassistant/components/signalmessenger/notify.py

```diff
@@ -0,0 +1,73 @@
+"""Signal Messenger for notify component."""
+import logging
+
+import voluptuous as vol
+from pysignalclirestapi import SignalCliRestApi
+from pysignalclirestapi import SignalCliRestApiError
```

Comment:
The two imports from `pysignalclirestapi` can be done on the same line.

I recommend the isort tool to automatically sort the imports.